android/src/main/java/io/customer/reactnative/sdk/CustomerIOReactNativeModule.kt

```diff
@@ -65,7 +59,7 @@ class CustomerIOReactNativeModule(
         val packageConfig = packageConfiguration?.toMap()
 
         try {
-            customerIO = CustomerIOReactNativeInstance.initialize(
+            val newInstance = CustomerIOReactNativeInstance.initialize(
```

Comment:
Will `newInstance` be garbage collected when the function exits? 

Previously the code saved to property `customerIO` and didn't get GCed.

Reply:
🤔 I think there won't any affect on it. As `CustomerIOReactNativeInstance.initialize` here is returning instance returned by `CustomerIO.Builder`, which is eventually static `instance` held by `CustomerIO`.

Reply:
Agreed. 

Is there a reason to include `val newInstance = `?

Reply:
Only for the following [line](https://github.com/customerio/customerio-reactnative/pull/155/files/6e243bea09f5a80bdecbf090a252352d3a2796a7#diff-215c64b88207fb848c32de218ab873cba8b7d8a13dd90521637e61696f057875R76):

```
val lifecycleCallbacks = newInstance.diGraph.activityLifecycleCallbacks
```

We can use `CustomerIO.instance().diGraph.activityLifecycleCallbacks` as well, but since the instance is already being returned, I think it is better to use the returned one.